src/mumble/Plugins.h

```diff
@@ -62,8 +62,8 @@ class Plugins : public QObject {
 	void clearPlugins();
 	int iPluginTry;
 	QMap< QString, PluginFetchMeta > qmPluginFetchMeta;
-	QString qsSystemPlugins;
-	QString qsUserPlugins;
+	QString downloadPath;
+	QStringList loadPaths;
```

Comment:
I think `m_pluginSearchPath` would be a more fitting name

Reply:
`m_pluginSearchPaths`?

Reply:
yes